subnet_test.go

```diff
@@ -86,7 +86,6 @@ var subnetResponse = `
         "space": "space-0",
         "id": 34,
         "resource_uri": "/MAAS/api/2.0/subnets/34/",
-        "dns_servers": [],
```

Comment:
I've checked the output - and the problematic one *is* "null", so I've updated the test.